aspnetcore/docfx.json

```diff
@@ -66,7 +66,7 @@
         "**/tutorials/**/**.md": "tutorial"
       },
       "no-loc": {
-        "**/**.md": [ "toaster", "bowl", "toothbrush", "magnet", "oak" ]
+        "**/**.md": [ ".NET MAUI", "Mac Catalyst", "Blazor Hybrid", "Home" , "Privacy" , "Kestrel", "appsettings.json", "ASP.NET Core Identity", "cookie", "Cookie", "Blazor", "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", "Razor", "SignalR" ]
```

Comment:
Here's the magic.
<img width="190" alt="image" src="https://user-images.githubusercontent.com/3605364/171774558-5bb88aca-8939-4198-b749-9dc9190a6173.png">